### ISSUE TITLE
Delimiter aria label parts with ,

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#96046]
In some places with seperate parts of the aria label only with a whitespace and somewhere with a `,`. We should be consistent and use `,` everywhere. Reasons:
1) iOS and MacOS use commas for delimiting labels that VoiceOver should speak, in most cases I believe.
2) When screen reader faces a comma it slows down making the separation more distinguishable

Example: lets say I have a a/b/c/test.txt file which is modified and shown in the Git viewlet. The aria label should be 
`text.txt, modified, a/b/c`

Thanks @RedEyeDragon and @marcozehe for great feedback.

fyi @joaomoreno @bpasero @sandy081 @roblourens @jrieken since I might polish your labels.

**Comment by isidorn** [comment]
Fixed most cases that I could found. Other cases we will fix as we go. More importantly now we have a recommendation to use comma as a delimeter in aria labels.
